src/Simplex/Messaging/Agent/Protocol.hs

```diff
@@ -328,6 +329,7 @@ data ACommandTag (p :: AParty) where
   ERR_ :: ACommandTag Agent
   SUSPENDED_ :: ACommandTag Agent
   FRCVD_ :: ACommandTag Agent
+  FRCVERR_ :: ACommandTag Agent
```

Comment:
I think it should be just inside ERR with subtype

Reply:
discussed, to do - decide on where to include file id in event, possibly generalize conn id to entity id